main.tf

```diff
@@ -1,31 +1,110 @@
-data "google_secret_manager_secret_version" "monitoring_slack_token" {
-  project = var.project_id_slack_token
-  secret  = "monitoring-slack-token"
+data "google_secret_manager_secret_version" "slack_token" {
+  count   = var.slack_token_secret_project_id != "" && length(try(var.notification_channels.slack, [])) >= 1 ? 1 : 0
+  project = var.slack_token_secret_project_id
+  secret  = var.slack_token_secret_name
 }
 
-resource "google_monitoring_notification_channel" "clan_email" {
-  count        = var.clan_group_email != "" ? 1 : 0
-  project      = var.tribe_project_id
-  display_name = "Email channel for ${var.clan_name} clan"
+resource "google_monitoring_notification_channel" "slack" {
```

Comment:
Why won't you keep the resource name as `clan_slack_channels`? It will be less changes to do after this is merged

Reply:
( Same reason as below )